x/gov/keeper/internal_test.go

```diff
@@ -0,0 +1,8 @@
+package keeper
+
+import "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+func UpdateHooks(k *Keeper, h types.GovHooks) *Keeper {
```

Comment:
can you define this on the other test file instead?

Reply:
No, so the reason it's like this is that the `hooks` field in the keeper is unexposed, so it can only be accessed by the `keeper` package, not the `keeper_test` package.

And we don't want to add this function to the `keeper.go` file because that then it would be available to all other modules, which isn't what we want.  The trick is to put it in a file with the `_test` suffix but still in the `keeper` package, so it's only built during tests and thus accessible to other tests, but not other modules.

See:  https://stackoverflow.com/questions/24622388/how-to-test-a-unexported-private-function-in-go-golang

Reply:
makes sense! thanks for the clarification